rules/okta/okta_attempt_to_modify_or_delete_application_sign_on_policy.toml

```diff
@@ -0,0 +1,35 @@
+[metadata]
+creation_date = "2020/07/01"
+ecs_version = ["1.5.0"]
+maturity = "production"
+updated_date = "2020/07/01"
+
+[rule]
+author = ["Elastic"]
+description = """
+An adversary may attempt to modify or delete the sign on policy for an Okta application in order to remove or weaken an
+organization's security controls.
+"""
+false_positives = [
+    """
+    Consider adding exceptions to this rule to filter false positives if sign on policies for Okta applications are
+    regularly modified or deleted in your organization.
+    """,
+]
```

Comment:
This feels like a given to me, with the description we have. If you think it still adds value, then feel free to keep.
```suggestion
```

Reply:
This wording does align with what we've included for other rules and could be helpful to less-experienced users. I'll leave this conversation open and let the other reviewers provide their feedback. If we decide to remove this wording, we can reserve `false_positives` for specific behaviors/activity that can be exempted from the rule. 👍